sentry_sdk/hub.py

```diff
@@ -548,6 +549,9 @@ def start_transaction(
         sampling_context.update(custom_sampling_context)
         transaction._set_initial_sampling_decision(sampling_context=sampling_context)
 
+        profile = Profile(transaction, hub=self)
```

Comment:
Not 100% confident here but the same applies to the span object where the span has a reference to the hub and the hub has a reference to the span.

I believe the reason this is okay is because the hub only has a reference to the span inside a with clause and the reference is removed afterwards but again Im not 100% sure.

Reply:
Not 100% confident here but the same applies to the span object where the span has a reference to the hub and the hub has a reference to the span.

I believe the reason this is okay is because the hub only has a reference to the span inside a with clause and the reference is removed afterwards but again Im not 100% sure.

Reply:
ok, then it should be ok